wrappers/services.go

```diff
@@ -111,6 +111,15 @@ func StartServices(apps []*snap.AppInfo, inter interacter) (err error) {
 				return err
 			}
 
+			defer func(socketService string) {
```

Comment:
err is shadowed in line 123 below, so, this will not clean up if sysd.Start() fails, no?

Reply:
err in the defer is the one the function returned, unless it's captured by the defer. See https://play.golang.org/p/jWmGpAwn_Pi to play with this.